.github/workflows/docker-test.yml

```
@@ -2,33 +2,37 @@ name: Docker Build & Test
+      - ".dockerignore"
+      - "requirements.txt"
+      - "**.py"
   workflow_dispatch:
 
 jobs:
   build-and-test:
     runs-on: ubuntu-latest
 
     services:
+      web:
+        env_file:
```

actionlint (1.7.4)

unexpected key "env_file" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)